MobileSyncExplorerReactNative/js/ContactCell.js

```diff
@@ -25,30 +25,30 @@
  */
 
 import React from 'react';
-import { ListItem } from 'react-native-elements';
+import { ListItem, Icon } from 'react-native-elements';
 import ContactBadge from './ContactBadge';
 
 class ContactCell extends React.Component {
     render() {
-        let statusIcon;
+        let statusIcon
         if (this.props.contact.__local__) {
-            if (this.props.contact.__locally_updated__) statusIcon = {name: 'sync', color:'blue'};
-            if (this.props.contact.__locally_created__) statusIcon = {name: 'add', color:'green'};
-            if (this.props.contact.__locally_deleted__) statusIcon = {name: 'delete', color:'red'};
-            if (this.props.contact.__last_error__) statusIcon = {name: 'sync-problem', color: 'red'};
+            if (this.props.contact.__locally_updated__) statusIcon = (<Icon name='sync' color='blue'/>);
+            if (this.props.contact.__locally_created__) statusIcon = (<Icon name='add' color='green'/>);
+            if (this.props.contact.__locally_deleted__) statusIcon = (<Icon name='delete' color='red'/>);
+            if (this.props.contact.__last_error__) statusIcon = (<Icon name='sync-problem' color='red'/>);
         }
 
         const fullName = [this.props.contact.FirstName, this.props.contact.LastName].filter(x=>x).join(' ')
         const title = this.props.contact.Title;
         
-        return (<ListItem
```

Comment:
We have jumped from react native elements 0.19.1 to 3.4.2. List items are different.